benefits/oauth/hooks.py

```diff
@@ -64,6 +65,7 @@ def claims_verified_eligible(cls, request, claims_request, claims_result):
     )
     def claims_verified_not_eligible(cls, request, claims_request, claims_result):
         super().claims_verified_not_eligible(request, claims_request, claims_result)
+        session.update(request, oauth_authorized=False)
```

Comment:
Confusingly, this also needs to be `True`: it is not saying "user was verified as eligible", it is saying "user successfully signed in"

Reply:
Notice in the old code, we didn't have a path for setting it to `False`. It was always `True`.

Reply:
Ah, yep - thanks for the catch :pray:  Agreed that it's confusing - maybe `logged_in` would be a less confusing variable name since that's the method where it's ultimately used. Anyway, not going to worry about the naming in this PR.

Reply:
I ended up renaming the variable in https://github.com/cal-itp/benefits/pull/2783/commits/fc8e3cae9e5fff60c04f1fbef8b52cf4b2b80395 - was easy enough and cleaned up some code too